### ISSUE TITLE
Machine Learning Method Training Optimization

### ISSUE THREAD

**Issue by nolankurylo** [SENG499-S22-Company3/algorithm-2#27]
- Ensure that all features have the correct range of values
  - BUG: "Semester" feature currently only has Fall, Spring values and no Summer value
  
- Check if features are being preprocessed properly:
   - plot each feature against the target (capacity)
   - histograms, scatterplots
   - 
Optimize current Machine Learning Model and validating it
- Use MAE as metric
- Use MSE as loss function
- Try SVM, RF, XGBoost, etc

**Comment by nolankurylo** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Schedulater-629034391c13f00017e43cce/issues/SENG499-S22-Company3/algorithm-2/27?planning-poker) with ZenHub @BaraaDawod @DavidOkulski @Randon511 @Scault
